test.js

```diff
@@ -114,6 +114,66 @@ describe('Simple schemas', () => {
     expect(generate(schema)).toEqual(expected)
   })
 
+  it('String with const', function () {
+    const schema = {
+      type: 'string',
+      const: 'value'
+    }
+    const expected = `/**
+ * @typedef {"value"}
+ */
+`
+    expect(generate(schema)).toEqual(expected)
+  })
+
+  it('Number with const', function () {
+    const schema = {
+      type: 'number',
+      const: 23.3
+    }
+    const expected = `/**
+ * @typedef {23.3}
+ */
+`
+    expect(generate(schema)).toEqual(expected)
+  })
+
+  it('Integer with const', function () {
+    const schema = {
+      type: 'integer',
+      const: '42'
```

Comment:
Should this be `const: 42` instead? Likewise should the two `const`  below also be changed from a string to the value in question?

Reply:
You're absolutely right.
I changed it